aerosol_dynamics/physical_parameters.py

```diff
@@ -0,0 +1,29 @@
+"""
+Centralized location for physical parameters.
+"""
+
+from . import ureg
+
+# air viscosity in kg/m-s
+MEDIUM_VISCOSITY = 18.27e-6 * ureg.kg / ureg.m / ureg.s
+
+# mean free path of air in m
+MEAN_FREE_PATH_AIR = 65e-9 * ureg.m
+
+# Boltzmann's constant in m^2 kg s^-2 K^-1
+BOLTZMANN_CONSTANT = 1.380649e-21 * ureg.m ** 2 * ureg.kg / ureg.s / ureg.K
+
+# temperature in K
+TEMPERATURE = 300 * ureg.K
+
+# elementary charge in C
+ELEMENTARY_CHARGE_VALUE = 1.60217662e-19 * ureg.C
+
+# dielectric constant of air
+EPSI = 1.0005 # unitless
+
+# permittivity of free space in F/m
+EPS0 = 8.85418782e-12 * ureg.F / ureg.m

```

Comment:
Interesting, does this unit thing follow the legit conventions of uppercase, lowercase, etc. when it comes to units?

As an aside, I will also update some of the constants above to follow the latest iteration. Apparently, in 2019, the standard unit committee (or whatever they're called) changed the definitions of basic units to rely more on fundamental constants. So currently, there is an exact number for `BOLTZMANN_CONSTANT` and `ELEMENTARY_CHARGE_VALUE` and I think they're actually almost exactly the same as above (please change the power on k)

`BOLTZMANN_CONSTANT = 1.380649e-23`
`ELEMENTARY_CHARGE_VALUE = 1.602176634e-19`

If interested, see more about the politics of units: https://en.wikipedia.org/wiki/2019_redefinition_of_the_SI_base_units 

We should probably eventually redefine everything to go back to the very basic constants; that will take some time, but it will be an interesting exercise.

Reply:
Can you elaborate about changing the power on k? 
From my understanding, the values themselves have not changed, only the way they're defined. 
Pint should sort out most units issues for us. We'll have to do some testing to make sure certain combinations of units remains the way we want, but generally speaking Pint will take care of any conversions, and we can leave our constants in the form they are most commonly used/understood (e.g. no need to rewrite Coulomb as Ampere * seconds)

Reply:
It should be `BOLTZMANN_CONSTANT = 1.380649e-23...` not `BOLTZMANN_CONSTANT = 1.380649e-21 * ureg.m ** 2 * ureg.kg / ureg.s / ureg.K` --- that is, Boltzmann constant (k) has a power of -23, not -21. 

Sorry, I should have said the power of Boltzmann constant, not "k", when we have many ks around...

Reply:
(isn't that what the thing atop the 10 is called? the number 10^5 has a power of 5... or am I making this up?)

Reply:
"Power" is fine, "Exponent" might be clearer. I thought you meant ureg.K had the wrong exponent and was confused. Good catch on the number though